### PR TITLE
Change back install instruction to use github

### PR DESCRIPTION
**Pull request by jankatins**
See https://github.com/IRkernel/IRkernel/pull/327/ for the motivation.

### CODE REVIEW

**Comment by jankatins** [comment]
And just for the record: I feel filthy after changing that html code... Better the limitations of markdown and a plain jekyll build, than this ...

File: installation/index.html [hunk 1]
```diff
@@ -29,63 +29,12 @@
...
-		</div>
-		<h4>R packages</h3>
-		<p>Start <code>R</code> in the same terminal, and proceed as below.</p>
-		<p>You can install snapshot packages and the kernel spec via</p>
-		<pre class="r mdl-shadow--2dp"><code>install.packages(c('rzmq','repr','IRkernel','IRdisplay'),
-                 repos = c('http://irkernel.github.io/', getOption('repos')),
-                 type = 'source')</code></pre>
-		<p>To update your source installation, repeat above step.</p>
-	</div>
+		<p>We are currently in the process to submit our packages to CRAN. Until then, you can install via devtools with the following lines in an R console:</p>
```
Comment (takluyver) [inline installation/index.html]: Ditto with the wording

Comment (jankatins) [inline installation/index.html]: done

**Comment by flying-sheep** [comment]
well we can either have
1. good docs
   - with the current not-easily edited format
   - with a complex data-driven setup where text can be easily edited but new tabs and other structure are hard
2. worse docs

File: installation/index.html [hunk 2]
```diff
@@ -29,63 +29,12 @@
...
-		<pre class="r mdl-shadow--2dp"><code>install.packages(c('rzmq','repr','IRkernel','IRdisplay'),
-                 repos = c('http://irkernel.github.io/', getOption('repos')),
-                 type = 'source')</code></pre>
-		<p>To update your source installation, repeat above step.</p>
-	</div>
+		<p>We are currently in the process of submitting our packages to CRAN. Until then, you can install via devtools with the following lines in an R console:</p>
+		<pre class="r mdl-shadow--2dp"><code>install.packages(c('repr','pbdZMQ', 'devtools')) # repr is already on CRAN</code></pre>
+		<pre class="r mdl-shadow--2dp"><code>devtools::install_github('irkernel/IRdisplay')</code></pre>
+		<pre class="r mdl-shadow--2dp"><code>devtools::install_github('irkernel/IRkernel')</code></pre>
+		<pre class="r mdl-shadow--2dp"><code>IRkernel::installspec()</code></pre>
```
Comment (flying-sheep) [inline installation/index.html]: no… one code block for everything, please, not four.

the indentation will look like this, but that’s OK:

``` r
        <p>We are currently in the process of submitting our packages to CRAN. Until then, you can install via devtools with the following lines in an R console:</p>
        <pre class="r mdl-shadow--2dp"><code>install.packages(c('repr','pbdZMQ', 'devtools')) # repr is already on CRAN
devtools::install_github()
...
```

Comment (jankatins) [inline installation/index.html]: ok

**Comment by jankatins** [comment]
Re good vs bad docs: IMO the most important about docs is that they are current and accurate. At least I can say that I don't like to touch this repo because it's hard to start some work here because I always have to relearn how to change something. So the result is that we might have eye pleasing docs, but they are outdated (just look at the issues here and the "asked for FAQ entries"). MD is so much easier...

So maybe we can get some jekyll template which is not so bad for the eyes, but still easy to maintain?

**Comment by flying-sheep** [comment]
you’re right, that’s for the best. too bad the material design lite classes are so cumbersome to work with.

File: installation/index.html [hunk 3]
```diff
@@ -29,63 +29,12 @@
...
-                 repos = c('http://irkernel.github.io/', getOption('repos')),
-                 type = 'source')</code></pre>
-		<p>To update your source installation, repeat above step.</p>
-	</div>
+		<p>We are currently in the process of submitting our packages to CRAN. Until then, you can install via devtools with the following lines in an R console:</p>
+		<pre class="r mdl-shadow--2dp"><code>install.packages(c('repr','pbdZMQ', 'devtools')) # repr is already on CRAN
+devtools::install_github('irkernel/IRdisplay')
+devtools::install_github('irkernel/IRkernel')
+IRkernel::installspec()</code></pre>
+		<p>To update packages which are not yet on CRAN, you have to rerun <code>devtools::install_github(...)</code>code> lines.</p>
```
Comment (flying-sheep) [inline installation/index.html:37]: the end is broken: `</code>code> lines.</p>`

**Comment by flying-sheep** [comment]
oh man i assumed you ran this locally and checked if it worked. you removed the source tab accidentally!

i modified your commit into dc2aee221bc6ad5e99f7f3394b72e92ad565f978.

you’ll need to reset your repo to that commit.

**Comment by flying-sheep** [comment]
oh man i assumed you ran this locally and checked if it worked. you removed the source tab accidentally!

i modified your commit into baed2cc259c9bb492dd69fe81f845c3d19504f0d.

you’ll need to reset your repo to that commit.

in the future, you can rename your master branch to `gh-pages` and check http://janschulz.github.io/irkernel.github.io if it works.

**Pull request merged**